lib/motions/general-motions.coffee

```diff
@@ -201,7 +201,18 @@ class MoveToNextWord extends Motion
   wordRegex: null
   operatesInclusively: false
 
+  wordRegExp: (cursor, {includeNonWordCharacters}={}) ->
```

Comment:
No, I just copied cursor's regex function: https://github.com/atom/atom/blob/37cb6b6d9e2be35af51b4e7e812b833efc24ce11/src/cursor.coffee#L642.  My original intent was to merge the changes back to Atom.